dev/devicelab/lib/tasks/plugin_tests.dart

```diff
@@ -46,11 +57,19 @@ class FlutterProject {
   final Directory parent;
   final String name;
 
-  static Future<FlutterProject> create(Directory directory) async {
+  static Future<FlutterProject> create(Directory directory, String buildTarget, String language) async {
     await inDirectory(directory, () async {
-      await flutter('create', options: <String>['--org', 'io.flutter.devicelab', 'plugintest']);
+      await flutter('create', options: <String>[
+        '--org', 'io.flutter.devicelab',
+        '-${buildTarget[0]}', language,
```

Comment:
Correct, but slightly magical - just so happens that the abbreviation of `--android-language` (`-a`) is also the first letter of the build target 'apk'... :)